### ISSUE TITLE
Accordian - Figma Build Review

### ISSUE THREAD

**Issue by CillianHPE** [grommet/hpe-design-system#4379]

**Comment by julauxen** [comment]
Review done [here.](https://www.figma.com/design/5NUO23O2mDjiulO2H0S2Os/v2-Construction-Site?m=auto&node-id=3505-18090&t=3uqpa6uwSbNyUH0B-1) @CillianHPE

**Comment by CillianHPE** [comment]
@julauxen 

Changes are now made. Ready for another review round.

**Comment by julauxen** [comment]
Thank @CillianHPE, there's just one comment left, about the focus indicator [here.](https://www.figma.com/design/5NUO23O2mDjiulO2H0S2Os/v2-Construction-Site?m=auto&node-id=3559-26104&t=PxTHBYgIUGsastXo-1)

**Comment by CillianHPE** [comment]
@julauxen 

THanks that focus indicator issue should be fixed now.
